docs/source/multidc/gke.md

```diff
@@ -0,0 +1,159 @@
+
+## Create and configure a VPC network
+
+For the clusters to have Inter-Kubernetes networking, we will create a virtual network shared between all the instances, with dedicated subnets for each of the clusters. 
+To create the subnets manually, we will create the network in custom subnet mode.
+
+### Create the VPC network
+
+Run the below command to create the network:
+```console
+gcloud compute networks create scylladb --subnet-mode=custom
+```
+
+With the VPC network created, we will create a dedicated subnet with secondary CIDR ranges for their Pod and Service pools in each region which the clusters will reside in.
+
+### Create VPC network subnets
+
+To create a subnet for the first cluster in region `us-east1`, run the below command:
+```console
+gcloud compute networks subnets create scylladb-us-east1 \
+    --region us-east1 \
+    --network scylladb \
+    --range 10.0.0.0/20 \
+    --secondary-range cluster=10.1.0.0/16,services=10.2.0.0/20
+```
+
+To create a subnet for the second cluster in region `us-west1`, run the below command:
+```console
+gcloud compute networks subnets create scylladb-us-west1 \
+    --region us-west1 \
+    --network scylladb \
+    --range 172.16.0.0/20 \
+    --secondary-range cluster=172.17.0.0/16,services=172.18.0.0/20
+```
+
+``` warning::
+    It is required that the IPv4 address ranges of the subnets allocated for the GKE clusters do not overlap.
+```
+
+Refer to [Create a VPC-native cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips) and [Alias IP ranges](https://cloud.google.com/vpc/docs/alias-ip) in GKE documentation for more information about VPC native clusters and alias IP ranges.
+
+## Create GKE clusters
+
+With the VPC network created, we will create two VPC native GKE clusters in dedicated regions.
+
+We will need to set kubelet CPU Manager policy to static by providing a config file. Save the following configuration in a `systemconfig.yaml` file:
+```
+kubeletConfig:
+  cpuManagerPolicy: static
+```
+
+### Create the first GKE cluster
+
+Run the following command to create the first GKE cluster in the `us-east1` region:
+```console
+gcloud container clusters create scylladb-us-east1 \
+    --location us-east1-b \
+    --node-locations us-east1-b,us-east1-c \
+    --machine-type n1-standard-8 \
+    --num-nodes 1 \
+    --disk-type pd-ssd \
+    --disk-size 20 \
+    --image-type UBUNTU_CONTAINERD \
+    --system-config-from-file systemconfig.yaml \
+    --no-enable-autoupgrade \
+    --no-enable-autorepair \
+    --enable-ip-alias \
+    --network scylladb \
+    --subnetwork scylladb-us-east1 \
+    --cluster-secondary-range-name cluster \
+    --services-secondary-range-name services
+```
+
+Refer to [Deploying Scylla on GKE](../gke.md) page of the documentation for reference regarding the deployment of additional node pools, including the one dedicated for ScyllaDB nodes.
```

Comment:
It's a little bit confusing. We are linking to entire document where we provide a script and walkthrough. After executing above command we can't execute the script nor follow entire walkthrough. Would be good to point to particular paragraph.

Reply:
I had some issues to linking to a particular header before so I left it like that but I fixed it now, thx

Reply:
Also fixed it for EKS doc